types/tsqlstring/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for tsqlstring 1.0
+// Project: https://github.com/kylefarris/tsqlstring
+// Definitions by: Dong Joon Seo <https://github.com/srencat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
+
+export function _escapeString(val: any): string;
+export function _zeroPad(number: number | string, length: number): string;
+export function _convertTimezone(tz: string): number;
```

Comment:
those looks like private, do you intent to expose them?

Reply:
In the existing module, it was declared private, but I added this function because I thought it was useful for the use of sql, which is the basic purpose of the module.

Reply:
However, if you don't think the type definition of this private function is necessary, you can remove it!

Reply:
everyone first look into readme or take whatever intellisense gives. Readme does not list any of those. Module is older version of ES2015 class, no private fields, so convention was used I think. I'd personally skip those three from export, similary to constants you've missed in your export. Those look fully internal IMO.

Reply:
That's right! I'm sorry for think myself personally. Thx :)